source/chapters/setup.rst

```diff
@@ -42,18 +19,18 @@ for club, hobby, and radio DJs alike.
 .. _setup-laptop-only:
 
 Laptop Only
-===========
+-----------
 
-.. figure:: ../_static/Mixxx-200-Preferences-Soundhardware.png
+.. figure:: ../_static/Mixxx-111-Preferences-Soundhardware.png
    :align: center
-   :width: 80%
+   :width: 75%
    :figwidth: 100%
    :alt: Using Mixxx with your built-in sound card
    :figclass: pretty-figures
 
    Using Mixxx with your built-in sound card
 
-There is no need to buy any equipment other than a computer if you do not
+There is no need to buy any equipment other than a computer if you do not 
```

Comment:
There is no need to insert white spaces before line breaks. Sphinx will add them automatically in the build. This should be reverted on all occasions across the commit.